archives/messages.go

```diff
@@ -44,7 +44,7 @@ SELECT rec.visibility, row_to_json(rec) FROM (
 			WHEN status = 'E' THEN 'errored'
 			WHEN status = 'F' THEN 'failed'
 			WHEN status = 'S' THEN 'sent'
-			WHEN status = 'R' THEN 'resent'
```

Comment:
there shouldn't be any of these in the db currently

Reply:
Yes, I verified that already